dev/com.ibm.ws.webcontainer.servlet.3.1.factories/test/com/ibm/ws/webcontainer/webapp/config/ServletConfigMock.java

```diff
@@ -198,12 +198,14 @@ public static WebAppConfiguration mergeTestXmls(final int version, String webXml
             }
         });
 
+        WebContainer webContainer = new WebContainer();
+
         Class<WebContainer> clazz = (Class<WebContainer>) webContainer.getClass();
 
-        Method versionSetter = clazz.getDeclaredMethod("setVersion", ServiceReference.class);
+        Field versionSetter = clazz.getDeclaredField("loadedContainerSpecLevel");
 
         versionSetter.setAccessible(true);
-        versionSetter.invoke(webContainer, versionRef);
+        versionSetter.set(webContainer, version);
```

Comment:
This is a static field so you don't have to pass an object to the Field.set method.  I think it is more clear to pass null here to make that explicit.